src/bin/casr-cluster.rs

```diff
@@ -24,6 +24,31 @@ use std::process::Command;
 use std::process::Stdio;
 use std::sync::RwLock;
 
+/// Check whether some serde_json key array value is not null or empty
+///
+/// # Arguments
+///
+/// * `json` - serde_json
+///
+/// * `key` - array value key
+///
+/// * `path` - json file path
+///
+/// # Return value
+///
+/// bool
+fn is_not_empty_as_array(json: &serde_json::Value, key: &str, path: &Path) -> Result<bool> {
+    if let Some(array) = json.get(key) {
+        if let Some(array) = array.as_array() {
+            Ok(!array.is_empty())
+        } else {
+            bail!("Error while parsing {}. File: {}", key, path.display());
```

Comment:
Also strange to use function parameter 'path' only in error case.

Reply:
Fixed